### PR TITLE
Breaking up some of lefthand.dmi and righthand.dmi.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
## Description of changes
Deletes several unused states from lefthand/righthand.dmi.
Reworks various items to use the single icon system instead.

## Why and what will this PR improve
Someday I want to destroy righthand.dmi and lefthand.dmi and this is needed for that.

## Authorship
Myself.

## Changelog
Nothing player-facing.

### CODE REVIEW

File: code/modules/projectiles/guns/launcher/syringe_gun.dm [hunk 1]
```diff
@@ -17,8 +17,10 @@
 	. = ..()
 	underlays.Cut()
 	if(syringe)
-		underlays += image(syringe.icon, src, syringe.icon_state)
-		underlays += syringe.filling
+		var/mutable_appearance/MA = syringe.appearance
```
Comment (out-of-phaze) [inline code/modules/projectiles/guns/launcher/syringe_gun.dm:20]: Are pixel_x and pixel_y cleared? Would they even apply anyway?

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
-/obj/item/chems/hypospray //obsolete, use hypospray/vial for the actual hypospray item
+/obj/item/chems/hypospray
 	name = "hypospray"
 	desc = "A sterile, air-needle autoinjector for rapid administration of drugs to patients."
-	icon = 'icons/obj/syringe.dmi'
-	item_state = "hypo"
-	icon_state = "hypo"
+	icon = 'icons/obj/hypospray.dmi'
+	icon_state = ICON_STATE_WORLD
+	abstract_type = /obj/item/chems/hypospray
```
Comment (out-of-phaze) [inline code/modules/reagents/reagent_containers/hypospray.dm:10]: Isn't it convention for `abstract_type` to come first, i.e. before `name =`? That makes it clearer that it's abstract.

**Review by out-of-phaze — COMMENTED** [review]
Did you make sure to test that syringe filling overlays still work? Without IDB, it's hard to check that without checking out the branch and running it, which is kind of a pain.

File: code/modules/reagents/reagent_containers/hypospray.dm (hunk 2 above)
Comment (MistakeNot4892) [reply code/modules/reagents/reagent_containers/hypospray.dm:10]: I wasn't aware we had a convention for that

File: code/modules/projectiles/guns/launcher/syringe_gun.dm (hunk 1 above)
Comment (MistakeNot4892) [reply code/modules/projectiles/guns/launcher/syringe_gun.dm:20]: Fixed.

File: code/modules/reagents/reagent_containers/hypospray.dm (hunk 2 above)
Comment (MistakeNot4892) [reply code/modules/reagents/reagent_containers/hypospray.dm:10]: Per Discord, leaving it where it is, but I readded a comment.

File: code/modules/projectiles/guns/launcher/syringe_gun.dm (hunk 1 above)
Comment (out-of-phaze) [reply code/modules/projectiles/guns/launcher/syringe_gun.dm:20]:
```dm
	if(randpixel)
		pixel_z = randpixel //an idea borrowed from some of the older pixel_y randomizations. Intended to make items appear to drop at a character
```
Looks like pixel_z should also be cleared. :(
EDIT: Maybe pixel_w too, at this point.

**Review by out-of-phaze — CHANGES_REQUESTED** [review]

**Review by out-of-phaze — APPROVED** [review]

**Pull request merged**